build.gradle

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    id 'java'
+    id 'maven'
+}
+
+repositories {
+    mavenLocal()
+    maven {
+        url = 'https://repo.spring.io/snapshot'
+    }
+
+    maven {
+        url = 'https://repo.spring.io/milestone'
+    }
+
+    maven {
+        url = 'http://repo.maven.apache.org/maven2'
+    }
+}
+
+dependencies {
+    compile 'org.springframework.boot:spring-boot-starter-actuator:2.1.1.RELEASE'
```

Comment:
We should use the dependency management features in Spring Boot - no need for explicit versions in `build.gradle`.

Reply:
perhaps this is mentioned in https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-build-systems.html#using-boot-gradle